Packs/Core/ReleaseNotes/2_0_9.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Investigation & Response
+
+Fixed an issue where the `core-list-risky-users` and `core-list-risky-hosts` commands would fail when the XDR Identity Threat Module is disabled or the license was missing.
```

Comment:
```suggestion
Fixed an issue where the ***core-list-risky-users*** and ***core-list-risky-hosts*** commands would fail when the XDR Identity Threat Module was disabled or the license was missing.
```